processor/ratelimitprocessor/processor.go

```diff
@@ -94,6 +99,7 @@ func NewMetricsRateLimiterProcessor(
 	telemetrySettings component.TelemetrySettings,
 	strategy Strategy,
 	next func(ctx context.Context, metrics pmetric.Metrics) error,
+	inflight *int64,
```

Comment:
nit: just me bein weird, that we name the metrics concurrent and the variable inflight. 
Maybe to add a comment

```suggestion
	inflight *int64, // used to calculate concurrent requests
```

Reply:
Make sense, I just feel inflight is more straightforward for some reason 😂 Will add the comment! Thanks!